ocfweb/main/templates/main/staff-hours.html

```diff
@@ -8,30 +8,39 @@
         <h2>Have questions? Drop by for help from a friendly volunteer staffer!</h2>
         <p>OCF staff members hold regular drop-in staff hours to provide assistance with account issues or with OCF services. We're always happy to help troubleshoot account or service issues!</p>
         <p>Keep in mind the OCF volunteers sometimes have last-minute conflicts, so it's a good idea to check this page before coming in for cancellations.</p>
-        {% for staff_hour in staff_hours %}
-            <div class="hour {% if staff_hour.cancelled or staff_hour.day == today and lab_status.force_lab_closed %} cancelled {% endif %}">
-                <div class="title">
-                    <strong>{{staff_hour.day}}</strong>
-                    {{staff_hour.time}}
-                    {% if staff_hour.cancelled or staff_hour.day == today and lab_status.force_lab_closed %}
-                        <span class="cancelled-text">cancelled this week</span>
+        {% for staff_day in staff_hours %}
+            <div class = "day">
+                <div class="daytitle">
+                    <strong><big>{{staff_day.day}}</big></strong>
+                    {%if staff_day.holiday != None %}
```

Comment:
`{%if staff_day.holiday is not none %}`

Reply:
@vaibhavjayaraman To explain this, `is` checks object identity eg if they are the same thing, `==` checks equality. Since `None` is a singleton (there can only be one), `is`/`is not` is faster and easier to read.